src/idl_gen_kotlin.cpp

```diff
@@ -1299,19 +1317,22 @@ class KotlinGenerator : public BaseGenerator {
   }
   static void GenerateGetterOneLine(CodeWriter &writer, const std::string &name,
                                     const std::string &type,
-                                    const std::function<void()> &body) {
+                                    const std::function<void()> &body,
+                                    bool gen_experimental_unsigned_types = false) {
     // Generates Kotlin getter for properties
     // e.g.:
     // val prop: Mytype get() = x
     writer.SetValue("_name", name);
     writer.SetValue("_type", type);
+    MarkExperimentalUnsignedTypesAnnotation(gen_experimental_unsigned_types);
     writer += "val {{_name}} : {{_type}} get() = \\";
     body();
   }
 
   static void GenerateGetter(CodeWriter &writer, const std::string &name,
                              const std::string &type,
-                             const std::function<void()> &body) {
+                             const std::function<void()> &body,
+                             bool gen_experimental_unsigned_type = false) {
```

Comment:
Why getters and setters need to be aware of the annotation? Isn't the class itself annotated not enough?